### PR TITLE
77 added index.html.template and updated docs

### PR DESCRIPTION
**Pull request by kamerynB**

### CODE REVIEW

**Comment by clarkepeterf** [comment]
We'll also want to delete the index.html file, as that will be stored in a separate private repository and incorporated at build time

**Comment by kamerynB** [comment]
@clarkepeterf The index.html has been removed from git. I still have my local copy as it is now ignored by git.

File: client/.gitignore [hunk 1]
```diff
@@ -28,3 +28,6 @@ package-lock.json
 /src/samples
 .env
 /src/lib/pushClientEvent.ts
+/public/
+!/public/uv/
```
Comment (kamerynB) [inline client/.gitignore:32]: We're ok with not pushing up the UV config? This would impact the user's ability to get the IIIF viewer working.

Comment (clarkepeterf) [reply client/.gitignore:32]: This says to ignore the public folder, but DON'T ignore the UV config. (The UV config will still be in the repo)

Comment (kamerynB) [reply client/.gitignore:32]: Welp, you learn something knew everyday. Merging!

**Pull request merged**